lib/Config/Menu.php

```diff
@@ -96,7 +96,7 @@ public function setup() {
            . '|Raleway:400,400i,700,700i'
            . '|Permanent+Marker:400,400i,700,700i'
            . '|Pacifico:400,400i,700,700i';
-          echo '<!--[if !mso]><link href="https://fonts.googleapis.com/css?family=' . $fonts . '" rel="stylesheet"><![endif]-->';
+          echo '<!--[if !mso]><!-- --><link href="https://fonts.googleapis.com/css?family=' . $fonts . '" rel="stylesheet"><!--<![endif]-->';
```

Comment:
This code is rendered in the browser. I think we should remove the HTML condition completely from this place and and echo the link without the condition